crates/anvil/core/Cargo.toml

```diff
@@ -16,7 +16,10 @@ foundry-utils.workspace = true
 revm = { workspace = true, default-features = false, features = ["std", "serde", "memory_limit"] }
 
 alloy-primitives = { workspace = true, features = ["serde"] }
-ethers-core.workspace = true
+ethers-core = { workspace = true, features = ["optimism"], optional = true }
+ethers-contract = { workspace = true, features = ["optimism"], optional = true }
+ethers-providers = { workspace = true, features = ["optimism"], optional = true }
+ethers-middleware = { workspace = true, features = ["optimism"], optional = true  }
```

Comment:
I don't think we need all of these? and can keep ethers-core?

Reply:
some tests in anvil/core seem to fail without these, for example `cargo test --all-features --package anvil-core decode_multiple_network_txs`. Is there a different way to run those tests?